src/components/decoration/textBlocks.tsx

```diff
@@ -35,6 +35,7 @@ interface SectionHeaderProps
   stackEntries?: boolean;
   className?: string;
   rootClassName?: string;
+  smallOnMobile?: boolean;
```

Comment:
Does it make sense to change the text size on mobil for every use of this component? We should keep the text sizes consistent over all pages.